.dockerignore

```diff
@@ -1,5 +1,4 @@
 docker-compose.yml
 docker-compose-dev.yml
-Makefile
```

Comment:
Need to include `Makefile` in the Docker build because we use `make` to build `hubd` in Docker.